README.md

```diff
@@ -10,7 +10,7 @@ These checks run on a regular schedule, and send the results to the [Operations
 Engineering Reports] web application.
 
 [standards]: https://ministryofjustice.github.io/technical-guidance/#building-software
-[Operations Engineering Reports]: https://operations-engineering-reports.cloud-platform.service.justice.gov.uk/
+[operations engineering reports]: https://operations-engineering-reports.cloud-platform.service.justice.gov.uk/
```

Comment:
With lowercase this doesn't match the reference on line 10. Either the reference needs updating or this change needs reverting

Reply:
I tested the link on https://github.com/ministryofjustice/github-repository-standards/tree/less-standards it does still work, dependabot changed it, maybe it can be case insensitive in markdown / html ?